packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1986,3 +2117,199 @@ String _concatStrings({
     return nestedLabel;
   return '$thisString\n$nestedLabel';
 }
+
+/// Provides a way to specify the order in which semantic nodes are sorted.
+///
+/// [TranversalSortOrder] objects contain a list of sort keys in the order in
+/// which they are applied. They are attached to [Semantics] widgets in the
+/// widget hierarchy, and are merged with the sort orders of their parent
+/// [Semantics] widgets. If [SemanticsSortOrder.discardParentOrder] is set to
+/// true, then they will instead ignore the sort order from the parents.
+///
+/// Keys at the same position in the sort order are compared with each other,
+/// and keys which are of different types, or which have different
+/// [SemanticSortKey.name] values compare as "equal" so that two different types
+/// of keys can co-exist at the same level and not interfere with each other,
+/// allowing for sorting into groups.  Keys that evaluate as equal, or when
+/// compared with Widgets that don't have [Semantics], fall back to the default
+/// upper-start-to-lower-end geometric ordering.
+///
+/// Since widgets are globally sorted by their sort key, the order does not have
+/// to conform to the widget hierarchy.
+///
+/// This class takes either `key` or `keys` at construction, but not both. The
+/// `key` argument is just shorthand for specifying `<SemanticsSortKey>[key]`
+/// for the `keys` argument.
+///
+/// Example usage:
```

Comment:
I believe this should be "## Sample code". That way, travis will run the analyzer over this code snippet and we have some kind of guarantee that it doesn't go out of date.

Reply:
Done.

Reply:
It still seems to show up as "Example usage" instead of "## Sample code"?